iXBRLViewerPlugin/viewer/src/js/util.js

```diff
@@ -207,3 +210,20 @@ export function getIXHiddenLinkStyle(domNode) {
     }
     return null;
 }
+
+/**
+ * Transforms measure qname into title case label (or currency symbol, if applicable).
+ * @return {String} Measure Label
+ */
+
+export function measureLabel(report, measure) {
+    const qname = report.qname(measure);
+    if (qname.namespace === NAMESPACE_ISO4217) {
+        measure = i18next.t(`currencies:unitFormat${qname.localname}`, {defaultValue: qname.localname});
+    } else if (measure.includes(':')) {
+        measure = titleCase(measure.split(':')[1]);
+    } else {
+        measure = titleCase(measure);
```

Comment:
I don't think that we should change the casing of units.  We have some units that are distinguished only by case (e.g. `Gt` vs `GT` for metric vs imperial gigatonnes)